spec/features/user_stories_spec.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../../lib/takeaway.rb'
+require_relative '../../lib/menu.rb'
+require 'twilio-ruby'
+require 'dotenv'
+
+describe "FEATURE SPEC" do
+  let(:takeaway) { Takeaway.new(Menu.new, Calculator.new,
+  Twilio::REST::Client.new("01234", "56789")) }
+  let(:choice) { {beef:2, rolls:3} }
+  let(:default_menu) { {ribs: 3,
+        beef: 4,
+        rolls: 3,
+        chips: 2,
+        pies: 5}
+        }
+  let(:confirm) {"Thank you! Your order was placed "\
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.